src/models/README.md

```diff
@@ -5,6 +5,7 @@ This documentation includes the following System76 models:
 - [Adder WS (addw1)](addw1/README.md)
 - [Bonobo WS (bonw14)](bonw14/README.md)
 - [Darter Pro (darp6)](darp6/README.md)
+- [Galago Pro (galp5)](galp5/README.md)
```

Comment:
I've gone ahead and split them. This page is kind of redundant with the `SUMMARY.md` sidebar, and I thought it might make sense for it to be a single list here since both "Current Models" and "Previous Models" in the sidebar go to this single page, but I'm sure some people will find it easier to navigate with the list split consistently in both places.